examples/plot_partial_dependence_visualization_api.py

```diff
@@ -43,19 +43,18 @@
 tree.fit(X, y)
 mlp.fit(X, y)
 
-
 ##############################################################################
 # Plotting partial dependence for two features
 # ============================================
 #
-# We plot partial dependence curves for features "LSTAT" and "RM" for
+# We plot partial dependence curves for features "s1" and "s5" for
```

Comment:
I think it would be nice to explain what these variables are. The list can be found in https://www.openml.org/d/37  however I'm not sure about the mapping with our feature names. Taking variables that are easy to understand would be ideal.

Reply:
I agree, otherwise I'm happy with the changes.

Reply:
There looks to be a mapping between our variables to more descriptive features: https://www4.stat.ncsu.edu/~boos/var.select/diabetes.html